index.html

```diff
@@ -204,7 +205,32 @@ <h3>Example</h3>
     "lprNumber": "999-999-999",
```

Comment:
This seems to be incorrect, as does `filingLocation` ... where is the `permanentResidentCard` entry? Shouldn't some of this stuff go in the card data?